src/cirrus/package.py

```diff
@@ -0,0 +1,388 @@
+#!/usr/bin/env python
+"""
+package
+
+Implementation for the package command that handles helping set up and
+manipulate packages for use with cirrus.
+
+Commands:
+
+package init - Initialise a new repo with a basic cirrus.conf file
+  add the appropriate setup, manifest and requirements files
+
+package sublime-project - Assistant to set up a sublime project for a cirrus
+  managed package, including build rules for the local venv
+
+"""
+import sys
+import os
+import contextlib
+import inspect
+import pystache
+import ConfigParser
+
+from argparse import ArgumentParser
+from cirrus.logger import get_logger
+from cirrus.utils import update_version
+import cirrus.templates
+
+from cirrus.git_tools import (
+    branch,
+    push,
+    get_tags,
+    tag_release,
+    commit_files_optional_push,
+    get_active_branch
+)
+
+
+DEFAULT_HISTORY_SENTINEL = "\nCIRRUS_HISTORY_SENTINEL\n"
+LOGGER = get_logger()
+
+
+def build_parser(argslist):
+    """
+    build CLI parser and process args
+    """
+    parser = ArgumentParser(
+        description=(
+            'git cirrus package command:'
+            ' initialises cirrus for an existing git repo'
+        )
+    )
+    parser.add_argument('command', nargs='?')
+
+    subparsers = parser.add_subparsers(dest='command')
+    init_command = subparsers.add_parser('init')
+    init_command.add_argument(
+        '--repo', '-r',
+        dest='repo',
+        default=os.getcwd()
+    )
+    init_command.add_argument(
+        '--source-dir', '-s',
+        help="source code directory within package, assumes top level dir if not set",
+        dest='source',
+        default=None
```

Comment:
Yeah, the initial dir is the package directory, if you do eg -s src then it will add repo/src if None it wont add it